components/LyricsEditor.vue

```diff
@@ -0,0 +1,107 @@
+<script setup lang="ts">
```

Comment:
The lyrics for miracle are pretty long and don't fit on a whole screen. Then I need to scroll up all the time to get to the button